Common.mk

```diff
@@ -148,8 +148,8 @@ IMAGE_IMPORT_CACHE?=type=registry,ref=$(LATEST_IMAGE) type=registry,ref=$(subst
 
 BUILD_OCI_TARS?=false
 
-LOCAL_IMAGE_TARGETS=$(foreach image,$(IMAGE_NAMES),$(image)/images/amd64) $(if $(filter true,$(HAS_HELM_CHART)),helm/build,) 
-IMAGE_TARGETS=$(foreach image,$(IMAGE_NAMES),$(if $(filter true,$(BUILD_OCI_TARS)),$(call IMAGE_TARGETS_FOR_NAME,$(image)),$(image)/images/push)) $(if $(filter true,$(HAS_HELM_CHART)),helm/push,) 
+LOCAL_IMAGE_TARGETS=$(foreach image,$(IMAGE_NAMES),$(image)/images/amd64) $(foreach image,$(HELM_IMAGE_LIST),$(image)/helm/build)
```

Comment:
Doesnt need to change in this PR, but id really like to move the helm targets off the IMAGE_TARGETS list and instead probably just put them directly on the BUILD_TARGETS and RELEASE_TARGETS, or define a new HELM_TARGETS.  Its never really felt right to me for them to be included in images... if i run `make images`, am i expecting helm to also build?